internal/pkg/instance/instance_linux.go

```diff
@@ -47,6 +47,8 @@ type File struct {
 	Config []byte `json:"config"`
 	UserNs bool   `json:"userns"`
 	IP     string `json:"ip"`
+	LogErr string `json:"logErr"`
```

Comment:
I don't think we need to store these log paths within the instance configuration. Instead I would suggest calling `instance.GetLogFilePaths()` directly from `internal/app/singularity/instance_linux.go`

Reply:
Thanks for the welcome! 

If I remove the log paths from within the instance configuration, I would have to call `instance.GetLogFilePaths()` twice in `internal/app/singularity/instance_linux.go` which seems redundant: 
- Printing instance logs from CLI using the newly created -l (--logs) flag 
- Populating InstanceInfo for displaying instance list JSON format using the -j flag

To me, it seems that storing the log paths within the instance configuration fits the organization of the code. I'm more than willing to make the change, but I just want to ask why shouldn't we store the log paths within the instance configuration?

Reply:
The main reason behind the suggestion is because these paths don't _need_ to be stored on disk within the instance configuration json. They can be deduced directly from the directory structure within `~/.singularity/instances`.

After thinking about this more, I think your current approach is the best option as it does make sense to record all instance configuration data(like log paths) within the configuration json.